netbox_topology_views/api/views.py

```diff
@@ -153,6 +153,7 @@ def list(self, request):
                 group_locations=group_locations,
                 group_racks=group_racks,
                 group_id=group_id,
+                straight_cables=individualOptions.straight_cables,
```

Comment:
That is not the way we did it so far. Please be consistent. Otherwise the code is harder to read. In addition, this solution may make it impossible to override the individual option.

It is not wrong to optimize code in general, but this should be a seperate task and should be done as a whole.